ps_googleanalytics.php

```diff
@@ -847,6 +849,35 @@ public function hookdisplayBackOfficeHeader()
         }
     }
 
+    /**
+     * Hook called after order status change
+     * Used to cancel order after cancelling it
+     */
+    public function hookActionOrderStatusPostUpdate(array $params)
+    {
+        // If we have the order AND the new order status being set AND the order status is changing to CANCELLED
+        if (empty($params['id_order'])
+            || empty($params['newOrderStatus']->id)
+            || (int) $params['newOrderStatus']->id !== (int) Configuration::get('PS_OS_CANCELED')
+        ) {
+            return;
+        }
+	            
+        // We check if the refund was already sent to Google Analytics
+        $gaRefundSent = Db::getInstance()->getValue(
+            'SELECT id_order FROM `' . _DB_PREFIX_ . 'ganalytics` WHERE id_order = ' . (int) $params['id_order'] . ' AND refund_sent = 1'
+        );
+            
+        if ($gaRefundSent === false) {
+            // We refund it and set the "sent" flag to true
+            $this->context->cookie->ga_admin_refund = $ga_scripts . 'MBG.refundByOrderId(' . json_encode(['id' => $params['id_order']]) . ');';
```

Comment:
Where do you get the variable `$ga_scripts` ?
I tried your PR in a production environment with my own personal GA ID and I got this error with debug mode on :
![image](https://user-images.githubusercontent.com/49909275/85584843-cc991200-b63f-11ea-8c93-d0e865af2bdf.png)

Moreover, GA is complaining about a "No HTTP response detected" : 
![image](https://user-images.githubusercontent.com/49909275/85584993-ec303a80-b63f-11ea-8599-e63443662836.png)

Reply:
Hi @SimonGrn, my mistake, it was an empty unneeded variable.

It's complaining because no view is sent to GA, only events are sent. Try the cancel now and you will see an event sent successfully.

![refund](https://user-images.githubusercontent.com/6097524/85588332-e25c0680-b642-11ea-9a73-cc18bb0b2b67.JPG)

I added some notes to the readme, it confused me also at first. https://github.com/PrestaShop/ps_googleanalytics/blob/dev/Readme.md